x/crosschain/types/cctx.go

```diff
@@ -194,6 +194,7 @@ func (m *CrossChainTx) AddOutbound(
 	m.GetCurrentOutboundParam().EffectiveGasPrice = msg.ObservedOutboundEffectiveGasPrice
 	m.GetCurrentOutboundParam().EffectiveGasLimit = msg.ObservedOutboundEffectiveGasLimit
 	m.GetCurrentOutboundParam().ObservedExternalHeight = msg.ObservedOutboundBlockHeight
+	m.GetCurrentOutboundParam().ConfirmationMode = msg.ConfirmationMode
```

Comment:
THe Confirmation mode should be copied when a new outbound is added for a revert 
in the function `AddRevertOutbound`

Reply:
Similarly, for `newCmdCCTX` we should set the field for CMD cctx's too.

Reply:
> THe Confirmation mode should be copied when a new outbound is added for a revert
in the function AddRevertOutbound

I was thinking: can revert outbound have different confirmation mode than inbound? they are two separate observations by observers.

Reply:
> Similarly, for newCmdCCTX we should set the field for CMD cctx's too.
True.

Reply:
No, they can't, but we should still populate the field when creating a new set of OutboundParams. 
It should not be problem to leave it empty , but we have a lot of pointer usage in the codebase which might create issues.

Reply:
I think there are two things to consider, based on this thread and the one above 
- Can inbound and outbound have different confirmations 
- Can outbound and revert have different confirmations 

I think, based on my understanding the value that is assigned by zetacore for outbounds is merely a placeholder, the actual value would be provided by the `voteOutbound` msg.  If that's correct I think it would make sense to initialize the field with an expected value , and then overwrite it when the vote is added (instead of leaving it blank )

Reply:
The commit to use same confirmation mode when adding new `OutboundParams` object: https://github.com/zeta-chain/node/pull/3506/commits/6b99338fff908c75e9658f2a79362339aa44aa15